ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -162,3 +181,71 @@ Run Training Operator ODH Test
     IF    ${result.rc} != 0
         FAIL    ${TEST_NAME} failed
     END
+
+Prepare Codeflare E2E Test Suite
+    Log To Console    "Restarting kueue"
+    Restart Kueue
+
+    Log To Console    "Downloading compiled test binary odh"
+    ${result} =    Run Process    curl --location --silent --output odh ${DISTRIBUTED_WORKLOADS_RELEASE_ASSETS}/odh && chmod +x odh
+    ...    shell=true
+    ...    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0
+        FAIL    Unable to retrieve odh compiled binary
+    END
+    Create Directory    %{WORKSPACE}/codeflare-odh-logs
+    Log To Console    "Retrieving user tokens"
+    ${common_user_token} =    Generate User Token    ${NOTEBOOK_USER_NAME}    ${NOTEBOOK_USER_PASSWORD}
+    Set Suite Variable    ${NOTEBOOK_USER_TOKEN}   ${common_user_token}
+    Log To Console    "Log back as cluster admin"
+    Login To OCP Using API    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}
+    RHOSi Setup
+
+Teardown Codeflare E2E Test Suite
+    Log To Console    "Log back as cluster admin"
+    Login To OCP Using API    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}
+    Log To Console    "Removing test binaries"
+    ${result} =    Run Process    rm -f odh
```

Comment:
Nitpick - can be replaced by `Remove File        ${test_binary}`, see above.

Reply:
I have updated odh binary deletion flow accordingly, please review : https://github.com/red-hat-data-services/ods-ci/pull/1657/commits/86ac51b68bce6d7f3da5af72697865352b178296